mail/rspamd/Portfile

```diff
@@ -1,79 +1,96 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; truncate-lines: t; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           cmake 1.0
+PortGroup           cmake 1.1
 PortGroup           github 1.0
 
-github.setup        vstakhov rspamd 0.8.3
+github.setup        rspamd rspamd 1.9.2
 
-version             0.8.3
-revision            3
+revision            0
 categories          mail
 license             BSD
 maintainers         nomaintainer
 platforms           darwin
+supported_archs     noarch
+license             Apache-2
+homepage            http://rspamd.com
 description         Rspamd filtering system is created as a replacement of popular spamassassin \
                     spamd and is designed to be fast, modular and easily extendable system.
 long_description    ${description}  Rspamd core is written in C language using event driven \
                     paradigma. Plugins for rspamd can be written in lua. Rspamd is designed to \
                     process connections completely asynchronous and do not block anywhere in code.
-homepage            http://rspamd.com
 
-if {${github.version} eq ${version}} {
-    master_sites        http://rspamd.com/downloads
-    use_xz              yes
-    checksums           rmd160  dc43a24c9eef555bd390ab31c4aa1deb8c5e7704 \
-                        sha256  20f788b08406fdaa8148d726e7ba18940108afff2d48b4f47726e3f2a6a88daf
-} else {
-    fetch.type          git
-    post-fetch {
-        system -W ${worksrcpath} "git submodule update --init"
-    }
-}
+# Necessary to address this issue: https://trac.macports.org/ticket/47197
+cmake.out_of_source no
 
-depends_build-append port:pkgconfig
+checksums           rmd160  a8dd52a6ede4c52289b23a69d3ea80602e46696e \
+                    sha256  dd1d8e9cdbb1c870be3c1bfee57a6894b6f8174595bc0684212a7c1afa4439f0 \
+                    size    4460935
 
-depends_lib         port:libevent \
+## Add ragel dependence after this issue resolved https://trac.macports.org/ticket/58321#comment:6
+## In the meantime, roll back to 6.10
+## git clone --single-branch https://github.com/macports/macports-ports.git
+## cd macports-ports/
+## git checkout 4b72f12e19e746d94db029eb72d78169e1bf7649
+## cd lang/ragel
+## sudo port install
+depends_build-append port:pkgconfig port:luajit
+## Uncomment after port:ragel is fixed
+# depends_build-append port:pkgconfig port:luajit port:ragel
```

Comment:
This PR should remain on hold and WIP until https://github.com/macports/macports-ports/pull/4218 is merged, at which point the above comments should be removed and the dependency on rage restored.

Reply:
rage fix now merged.